src/libaktualizr/crypto/p11engine.cc

```diff
@@ -145,7 +145,25 @@ boost::filesystem::path P11Engine::findPkcsLibrary() {
 }
 
 PKCS11_SLOT* P11Engine::findTokenSlot() const {
-  PKCS11_SLOT* slot = PKCS11_find_token(ctx_.get(), wslots_.get_slots(), wslots_.get_nslots());
+  unsigned int i, nslot;
+  PKCS11_SLOT* slot = NULL;
```

Comment:
Please, use `nullptr` instead of NULL for sake of consistency with the rest of the code.
Also, use the uniform initializer `PKCS11_SLOT* slot{nullptr}`